src/DocumentFormat.OpenXml.Linq/Feature/IOpenXmlPartRootXElementFeature.cs

```diff
@@ -9,7 +9,7 @@ namespace DocumentFormat.OpenXml.Packaging
     /// <summary>
     /// Feature to allow access to <see cref="XElement"/> representation of a <see cref="OpenXmlPart"/>.
     /// </summary>
-    public interface IPartRootXElementFeature
```

Comment:
The value of keeping this public allows a contract that could be implemented differently for testing/custom implementation unknown to the library itself.

Reply:
I would not have any problem with a public interface. I just made it internal in case you wanted to sort out naming issues, for example.

I am fine with this name specifically (and am hoping that you agree), so that is not the issue. If we could make breaking changes in the next major release (should that be necessary at all), we could make it public right now. However, if it does not have to be public in the short term, we could make it public in the next release and "let it sink in". I'd leave that up to you.

Reply:
That all sounds good to me!